cmd/template/framework/files/makefile.tmpl

```diff
@@ -11,6 +11,14 @@ build:
 run:
 	@go run cmd/api/main.go
 
+# Create DB container
+docker-run:
+	@docker-compose up
+
```

Comment:
Thinking about this i believe the best way to handle this is the following:

docker-run:
        # this uses V2
	@docker compose up
        if fail:
            echo "this requires Docker V2 - use docker-compose up"
        else:
            do nothing continue with execution
            
 
I think this is the best path since this is going to use a non deprecated dependency (v1 docker-compose).